spec/features/interact_with_cart_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'spec_helper'
+
+feature "InteractWithCart", type: :feature do
+  scenario "An anonymous user adds a product to the cart" do
+    @product = FactoryGirl.create(:product, sku: 'SKU-1', name: 'RoR Shirt')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.